isolate_test.go

```diff
@@ -20,25 +20,44 @@ func TestIsolateTermination(t *testing.T) {
 	t.Parallel()
 	iso := v8go.NewIsolate()
 	defer iso.Dispose()
-	ctx := v8go.NewContext(iso)
-	defer ctx.Close()
 
-	err := make(chan error, 1)
+	if iso.IsExecutionTerminating() {
+		t.Error("expected no execution to be terminating")
+	}
 
-	go func() {
-		_, e := ctx.RunScript(`while (true) { }`, "forever.js")
-		err <- e
-	}()
+	var terminating bool
+	fooFn := v8go.NewFunctionTemplate(iso, func(info *v8go.FunctionCallbackInfo) *v8go.Value {
+		ctx := info.Context()
+		obj := ctx.Global()
+		val, _ := obj.Get("loop")
+		loop, _ := val.AsFunction()
```

Comment:
Loops like the loop function was passed in to simplify this, then you forgot to actually use it

```suggestion
		loop, _ := info.Args()[0].AsFunction()
```